package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.22.9",
+  "version": "0.22.10",
```

Comment:
@superKalo would it be better if we create a new minor tag here 0.23, because 0.22.9 > 0.22.10 (it's confusing)

Reply:
Fine by me, we have a couple of tags where we go beyond x.x.10 :)